myinfo.gemspec

```diff
@@ -19,6 +19,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'jwe', '~> 0.4'
   s.add_dependency 'jwt', '~> 2.2'
+  s.add_dependency 'jose', '~> 1.1.3'
+  s.add_dependency 'pry-rails'
```

Comment:
this should not be a dependency of this gem, I believe this is for you development purpose right?